src/spetlr/etl/loaders/UpsertLoaderStreaming.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+
+from pyspark.sql import DataFrame
+
+from spetlr.etl import Loader, dataset_group
+from spetlr.etl.loaders import UpsertLoader
+from spetlr.etl.loaders.stream_loader import StreamLoader
+from spetlr.tables import TableHandle
+
+
+class UpsertLoaderStreaming(Loader):
```

Comment:
Suggestion: Get rid of this class and built the upsert functionality into the `StreamLoader` class